[DATA] Add your Profile

Are you new to **Open Source**. Maybe I can help you.

Join the amazing Open Source world by contributing to this, your first project. Add your profile to **𝕏 Profile**

Need Assistance, check out Readme file, it will guide you through out. Also you can make a comment below this, I will try to answer back as soon as possible

Best of luck with the new journey
https://xprofile.hellofaizan.me/

@hellofaizan After cloning the project in my local and running command "npm i" then "npm run dev", it's showing this error :

error - ./node_modules/@react-aria/grid/node_modules/@internationalized/date/dist/import.mjs:1:0
Module not found: Can't resolve '@swc/helpers/_/_class_private_field_init'

Import trace for requested module:
./node_modules/@react-aria/grid/node_modules/@react-aria/i18n/dist/real-module.mjs
./node_modules/@react-aria/grid/node_modules/@react-aria/i18n/dist/import.mjs
./node_modules/@react-aria/grid/dist/import.mjs
./node_modules/@react-aria/table/dist/module.js
./node_modules/@nextui-org/react/esm/table/table.js
./node_modules/@nextui-org/react/esm/table/index.js
./node_modules/@nextui-org/react/esm/index.js
./src/pages/_app.js

https://nextjs.org/docs/messages/module-not-found

What do next can you please suggest?

--- a/data/users.json
+++ b/data/users.json
@@ -68,5 +68,13 @@
         "github": "MannuVilasara",
         "banner_color": "#ff1493",
         "about": "Mannu | 16yo from India currenty working in cyber security | Chess, Anime, Coffee | Cats 🐈‍⬛ | Community Manager (Moderator at DEV communtiy)"
+    },
+
+    {
+        "username": "prabhurupe",
+        "name": "Rupesh Prabhu",
+        "github": "rupeshprabhu",
+        "banner_color": "#00dbda",
+        "about": "Rupesh Prabhu | Frontend Developer currently in India | Loves to review and understand technology"
     }
 ]

